styles/Page.module.scss

```diff
@@ -9,8 +9,89 @@
   max-width: 800px;
 }
 
-.centerContent {
-  text-align: center;
+.searchBar * {
+  display: block;
+}
+
+.searchBar label {
+  font-weight: 700;
+  color: #5A5A5A;
+}
+
+.searchBar input{
```

Comment:
I think input by default does not inherit font family - can you add `font-family: "DM Sans", sans-serif` and `font-style: 1rem`

Reply:
Good catch! :)